docs/source/install.rst

```diff
@@ -1,9 +1,21 @@
 Install
 =======
 
+Prerequisites
+-------------
+
+UCX depends on the following system libraries being present: ``libibverbs``,
```

Comment:
With the topological distance feature that was added to ucx-py, I noticed I also needed to install the `libhwloc` package. Is that already included in one of these packages?

Reply:
Yeah @quasiben is doing that in PR ( https://github.com/rapidsai/ucx-py/pull/347 ). Will rebase after that lands.

Reply:
Have merged in Ben's branch. `libhwloc` is a dependency of the `ucx-py` package. So should get pulled in automatically when installing that. We also have it listed in the build from source section below.